roles/openstack-stack/README.md

```diff
@@ -0,0 +1,9 @@
+# Role openstack-stack
```

Comment:
Let's merge this role with openshift-openstack and use include_role.

Reply:
Oops, we already do that, I just forgot to remove the old role.